hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/events/EventWatcher.java

```diff
@@ -157,13 +146,14 @@ protected synchronized void handleCompletion(COMPLETION_PAYLOAD
     }
   }
 
-  private synchronized void handleTimeout(EventPublisher publisher,
+  private synchronized Void handleTimeout(EventPublisher publisher,
```

Comment:
I didn't get this part, what is the impact of changing void-> Void and returning null

Reply:
Callback as getting registered has return type is Callback with "Void" Object, so this gives error if return type mismatch,

Syntax for acquire: `public synchronized Lease<T> acquire(
      T resource, Callable<Void> callback)`

Below is error if we keep void:
`Bad return type in lambda expression: void cannot be converted to Void`

Reply:
Got it , looks like autoboxing won't happen for void type. Thanks for explaining